addons/mail/models/mail_thread.py

```diff
@@ -2985,6 +2985,7 @@ def _message_auto_subscribe_notify(self, partner_ids, template):
             model_description = self.env['ir.model']._get(record._name).display_name
             values = {
                 'object': record,
+                'company': self.env.company,
```

Comment:
I am unsure this is the right company (hello MC, automated actions without company, ...). As we assign an user, shouldn't we use the user's company in the template ? (aka user user.company_id.color) ?

Reply:
Up, I am mostly sure this is a bad idea. Why not using the company of the record (if defined), or computing values before hand ?